packages/sanity/src/structure/panes/document/documentPanel/header/DocumentPanelHeader.tsx

```diff
@@ -84,9 +78,6 @@ export const DocumentPanelHeader = memo(
     const contextMenuNodes = useMemo(() => menuNodes.filter(isNotMenuNodeButton), [menuNodes])
     const showTabs = views.length > 1
 
-    // Subscribe to external timeline state changes
-    const rev = useTimelineSelector(timelineStore, (state) => state.revTime)
```

Comment:
This was removed from the Header and moved to [ `ChangesInspector` ](https://github.com/sanity-io/sanity/pull/7462/files#diff-098905f8d208d0d802ad026099778cfe44fe8c2d76b3ec7cd55e18ebf9294aa9R88)